### ISSUE TITLE
[Luscious] Connector not working

### ISSUE THREAD

**Issue by ShinryuAspect** [manga-download/hakuneko#6598]
### Is there an existing issue for this connector?

- [X] I have searched the existing issues

### Did you read the troubleshooting guide ?

- [X] I read it but nothing seems related

### Is the website of the connector working properly ?

- [X] Yes i'm able to see the manga within my standard browser

### Make sure to try Nightly Build

- [X] Yes nightly build didn't fix the issue

### What kind of issue are you encountering

Manga list isn't refreshing

### Describe the bug

Apparently, Luscious seems to have changed the URL itself and it's API URL, so the connector points to the previous ones.
I've tried adding the corrected .mjs with a plugin by copying the original Luscious.mjs from the Github and editting it. While that lets me search un Luscious posts, the final list it's incomplete, displaying around 5900 results while there are around 95000 results according to Luscious itself (the spinning update button is idle, so I assume it finished querying the results).
In addition, while perfoming clipboard-paste on the added .mjs, it does nothing while sending an error through the console.
I hoped I've stated any relevant information and, if needed, feel free to ask me any further details needed.

### To Reproduce

6.1.7
1. Name of connector: Luscious
2. Name of manga (if connector update is working): Update not working
3. Name of chapter (if manga update is working): Update not working
4. Copy any relevant error message that appears in a popup or (advanced) within the console (press F12)
![imagen_2023-12-18_164834752](https://github.com/manga-download/hakuneko/assets/36446897/281c38f1-290d-4013-a7ee-1c5671c53bad)
[cache-1702914553534.log](https://github.com/manga-download/hakuneko/files/13705732/cache-1702914553534.log)

Edited 6.1.7
1. Name of connector: Luscious
2. Name of manga (if connector update is working): [Little-Red Parts (1&2) - pokilewd](https://members.luscious.net/albums/littlered-parts-12_417389/)
3. Name of chapter (if manga update is working): Update not working
4. Copy any relevant error message that appears in a popup or (advanced) within the console (press F12)
![imagen_2023-12-18_172737592](https://github.com/manga-download/hakuneko/assets/36446897/0ddba85a-b2c0-4994-9b34-93abc80bd315)
[cache-1702917072862.log](https://github.com/manga-download/hakuneko/files/13706172/cache-1702917072862.log)

**Comment by MikeZeDev** [comment]
Nah they are just making graphql requests using GET and not POST (which is uglyAF cause request url is HUGE)

Our code still use POST and maybe outdated request

**Comment by ShinryuAspect** [comment]
Forgive me for "reopening" this, but I think I've found why the clipboard pasting is failing. I've made a console.log of the DOM it creates via the createDOM method in Connector.mfs and it seems to be landing onto the login screen, so the 'main h1.album-heading' selector passed to the fetchDOM method fails to locate any as those are classes found on the album themselves, not on the login page.

EDIT: let's just pretend that I knew that members.luscious & luscious are 2 completele separate domains and that was the reason why it waso leanding onto the Login screen, as I was clipboard-pasting a members.luscious link.
